kalite/coachreports/tests/api_tests.py

```diff
@@ -1,41 +0,0 @@
-import json
```

Comment:
Pass the tests by removing them, eh? I like it. It seems that the endpoint is genuinely unused though. +1